client.go

```diff
@@ -448,10 +433,6 @@ func (c *Client) PositionECEF() *PositionECEF {
 }
 
 func (c *Client) send(ctx context.Context, message Message) error {
-	deadline, _ := ctx.Deadline()
```

Comment:
no more deadline/timeout?

Or is it to be built in at a level above, when we provide the port (/reader)?

Reply:
this deadline is set on the connection in case of the UDP emulator